django_reverse_admin/__init__.py

```diff
@@ -310,13 +313,9 @@ def _changeform_view(self, request, object_id, form_url, extra_context):
                                       )
         media = self.media + adminForm.media
 
-        inline_admin_formsets = []
-        for inline, formset in zip(self.get_inline_instances(request), formsets):
-            fieldsets = list(inline.get_fieldsets(request))
-            inline_admin_formset = helpers.InlineAdminFormSet(inline, formset, fieldsets,
-                                                              readonly_fields=readonly_fields)
-            inline_admin_formsets.append(inline_admin_formset)
-            media = media + inline_admin_formset.media
+        inline_admin_formsets = self.get_inline_formsets(request, formsets, self.get_inline_instances(request), obj)
```

Comment:
Is this change necessary? It's breaking the delete flow in reverse admin for reasons that I cannot yet understand

Reply:
Actually, i just confirmed that it's fine. Thanks for your contribution. Merging the PR